server/src/test/java/io/druid/server/coordination/ZkCoordinatorTest.java

```diff
@@ -191,6 +257,114 @@ public void tearDown() throws Exception
     tearDownServerAndCurator();
   }
 
+  /**
+   * Steps:
+   * 1. removeSegment() schedules a delete runnable that deletes segment files after 1 second,
```

Comment:
"after 1 second" is not true anymore and shoud be removed from the comment... right?